File: Aula4/ComputationalThinkingAula4/src/exercicio7.java
import java.util.Scanner;

public class exercicio7 {

	public static void main(String[] args) {
		// Entrada: Usuário digitará a altura (h) e o raio (R) da lata de óleo.
		//Processamento: Calcular o volume (V) cúbico da lata com a fórmula V = 3,14 * (R^2) * h.
		//Saída: Exibir o volume cúbico(³) de uma lata de óleo.
		
		float h, R, volume;
		
		Scanner leitor = new Scanner(System.in);
			System.out.println("Insira a altura e o raio da sua lata de óleo respectivamente");
			h = leitor.nextFloat();
			R = leitor.nextFloat();
			
			leitor.close();
			
			volume = (float) ((3.14*(R*R)) * h);
			
			System.out.println("O volume cúbico da sua lata de óleo é " +volume);

	}

}

File: aula5e6/exerciciosPython/exercicio7.py
'''Calcula o volume cúbico de uma lata de óleo. O usuário
informa a altura e o valor de r. O programa calcula através
da fórmula VOLUME = 3.14 * (R * R) * ALTURA'''

h = float(input("Insira a altura da sua lata de óleo: "))
R = float(input("Insiro raio da sua lata de óleo: "))

volume = (3.14*(R*R)) * h

print("O volume cúbico da sua lata de óleo é {}".format(volume))
